### ISSUE TITLE
TwitterAuthService is null in DefaultTwitterAuthDao

### ISSUE THREAD

**Issue by keibro** [splix/grails-spring-security-twitter#16]
Running Grails 2.2.2, my custom TwiterrAuthService is not autowired to DefaultTwitterAuthDao.

Similar to the process followed in the Grails Spring Security Facebook plugin, should the following code be added to the _afterPropertiesSet_ method (note - it appears that the applicationContext is not yet available at this point either):

```
    if (!twitterAuthService) {
        if (grailsApplication.mainContext.containsBean('twitterAuthService')) {
            log.debug("Use provided twitterAuthService")
            twitterAuthService = grailsApplication.mainContext.getBean('twitterAuthService')
        }
    }
```

**Comment by onats** [comment]
Hi,

Confirmed issue. Is there a workaround to this? I need to use custom methods too.

**Comment by splix** [comment]
Sorry, missed this issue. Fixed in version 0.5.3
